modules/op-release-notes-1-11.adoc

```diff
@@ -0,0 +1,194 @@
+
+
+[id="known-issues-1-11_{context}"]	
+== Known issues	
+
+* Setting the `prune-per-resource` boolean field does not delete `PipelineRun` or `TaskRun` resources, if they were not part of any pipeline or task. 
+
+* Tekton CLI does not show logs of the `PipelineRun` resources that are created by using resolvers.
+
+* When you filter your pipeline results based on the query `order_by=created_time+desc&page_size=1`, you get zero records without any `nextPageToken` in the output. 
+
+
+[id="fixed-issues-1-11_{context}"]	
+== Fixed issues	
+
+* Before this update, {pac} failed to create a `PipelineRun` resource when it would detect the `generateName` field in the `PipelineRun` CR. With this update, {pac} supports providing the `generateName` field in the `PipelineRun` CR.
+
+* Before this update, when you created a `PipelineRun` resource from the web console, all annotations would be copied from the pipeline, causing issues for the running nodes. This update now resolves the issue.
+
+* This update fixes the `tkn pr delete` command for the `keep` flag. Now, if the value of the `keep` flag is equal to the number of the associated task runs or pipeline runs, then the command returns the exit code `0` along with a message.
+
+* Before this update, the Tekton Operator did not expose the performance configuration fields for any customizations. With this update, as a cluster administrator, you can customize the following performance configuration fields in the `TektonConfig` CR based on your needs:
+** `disable-ha`
+** `buckets`
+** `kube-api-qps`
+** `kube-api-burst`
+** `threads-per-controller` 
+
+* This update fixes the remote bundle resolver to perform a case-insensitive comparison of the `kind` field with the `dev.tekton.image.kind` annotation value in the bundle. 
+
+* Before this update, pods for remote resolvers were terminated because of insufficient memory when you would clone a large Git repository. This update fixes the issue and increases the memory limit for deploying remote resolvers. 
+
+* With this update, task and pipeline resources of `v1` type are supported in remote resolution.
+
+* This update reverts the removal of embedded `TaskRun` status from the API.  The embedded `TaskRun` status is now available as a deprecated feature to support compatibility with older versions of the client server.
+
+* Before this update, all annotations were merged into `PipelineRun` and `TaskRun` resources even if they might not be required for the execution. With this update, when you merge annotations into `PipelineRun` and `TaskRun` resources, the `last-applied-configuration` annotation is skipped.
+
+* This update fixes a regression issue to not validate a skipped task result in pipeline results. For example, if a `PipelineTask` resource referenced by the pipeline result was skipped, the pipeline result will not be emitted and the `PipelineRun` execution will not fail due to a missing result.
```

Comment:
Too much passive voice. Very confusing. Needs rewrite for complicity and clarity.